code/otel.AppHost/Program.cs

```diff
@@ -3,22 +3,23 @@
 var builder = DistributedApplication.CreateBuilder(args);
 
 var loki = builder.AddContainer("loki", "grafana/loki", "2.9.5")
-    .WithEndpoint(containerPort: 3100, hostPort: 3100, name: "http", scheme: "http")
-    .WithEndpoint(containerPort: 9096, hostPort: 9096, name: "grpc", scheme: "http")
+    .WithEndpoint(targetPort: 3100, port: 3100, name: "http", scheme: "http")
+    .WithEndpoint(targetPort: 9096, port: 9096, name: "grpc", scheme: "http")
     .WithBindMount("../config/loki.yml", "/etc/loki/local-config.yaml")
-    .WithVolumeMount("loki", "/data/loki")
+    .WithBindMount("../../data/loki", "/data/loki")

```

Comment:
I would rather revert this back to volume mount instead of bind mount.
I understand that in regular setup you want to export the data to the host for backup reasons, but this setup forces the the user that clones the repo to understand that he needs to create the directories.
If not, the apphost will throw exceptions at startup.

Reply:
Hey no worries, can you tell I was trying to shoe horn this into something I was using myself :) 

I've addressed both of these in follow up commits.

Reply:
Great thanks!